plugins/jira-plugin/src/main/java/io/qameta/allure/jira/JiraExportPlugin.java

```diff
@@ -74,124 +72,71 @@ public void aggregate(final Configuration configuration,
-                .orElse(new ExecutorInfo());
-    }
-
-    private Statistic getStatistic(final List<LaunchResults> launchesResults) {
-        final Statistic statistic = new Statistic();
-        launchesResults.stream()
-                .map(LaunchResults::getAllResults)
-                .flatMap(Collection::stream)
-                .forEach(statistic::update);
-        return statistic;
-    }
-
-    private JiraLaunch exportLaunchToJira(final JiraService jiraService, final JiraLaunch launch) {
+    private List<JiraExportResult> exportLaunchToJira(final JiraService jiraService,
+                                                      final JiraLaunch launch,
+                                                      final List<String> issues) {
         try {
-            final JiraLaunch created = jiraService.createJiraLaunch(launch);
-            LOGGER.info(String.format("Allure launch '%s' synced with issues '%s' successfully",
-                    created.getExternalId(), created.getIssueKeys()));
+            final List<JiraExportResult> created = jiraService.createJiraLaunch(launch, issues);
+            LOGGER.info(String.format("Allure launch '%s' synced with issues  successfully%n",
+                    issues));
+            JiraExportUtility.handleFailedExport(created);
+            LOGGER.info(String.format("Results of launch export %n %s", created));
             return created;
         } catch (Throwable e) {
-            LOGGER.error(String.format("Allure launch sync with issue '%s' error", launch.getIssueKeys()), e);
+            LOGGER.error(String.format("Allure launch sync with issue '%s' error", issues), e);
             throw e;
         }
     }
 
-    private void exportTestResultToJira(final JiraService jiraService, final JiraTestResult testResult) {
+    private void exportTestResultToJira(final JiraService jiraService,
+                                        final JiraTestResult jiraTestResult,
+                                        final TestResult testResult) {
+
+        if (!jiraTestResult.getTestCaseId().equals(testResult.getUid())) {
+            return;
+        }
+
         try {
-            final JiraTestResult created = jiraService.createTestResult(testResult);
-            LOGGER.info(String.format("Allure test result '%s' synced with issue '%s' successfully",
-                    created.getId(),
-                    created.getIssueKeys()));
+            final List<String> issues = testResult.getLinks().stream()
+                    .filter(JiraExportUtility::isIssueLink)
+                    .map(Link::getName)
+                    .collect(Collectors.toList());
+
+            final List<JiraExportResult> created = jiraService.createTestResult(jiraTestResult, issues);
+            JiraExportUtility.handleFailedExport(created);
+            created.forEach(creation -> {
+                        LOGGER.info(String.format("Allure test result '%s' synced with issue '%s' successfully",
```

Comment:
May be debug? 100 issues -> 100 info notes

Reply:
done while changing behaviour of handleFailures method